src/include/network/routines.rb

```diff
@@ -712,68 +712,44 @@ def GetAllInterfaces
       end
     end
 
-    # Wrapper to call 'ip link set up' with the given interface
+    # Calls wicked ifup if it is the currenct backend, set the link up
+    # otherwise
     #
-    # @param [String] name of interface to 'set link up'
-    def SetLinkUp(dev_name)
-      log.info("Setting link up for interface #{dev_name}")
-      Run("ip link set #{dev_name} up")
+    # @param [String] name of interface to put down
+    def SetIfaceUp(dev_name)
+      log.info("Setting interface #{devname} up")
+      Run("ifup #{dev_name} up")
     end
 
-    # Wrapper to call 'ip link set down' with the given interface
+    # Calls wicked ifdown if it is the current backend, set the link down
+    # otherwise
     #
-    # @param [String] name of interface to 'set link down'
-    def SetLinkDown(dev_name)
-      log.info("Setting link down for interface #{dev_name}")
-      Run("ip link set #{dev_name} down")
+    # @param [String] name of interface to put down
+    def SetIfaceDown(dev_name)
+      log.info("Setting interface #{devname} down")
+      Run("ifdown #{dev_name}")
     end
 
-    # Tries to set up the link of all available interfaces
+    # Tries to set all interfaces up
     #
     # @return [boolean] false if some of interfaces cannot be set up
-    def SetAllLinksUp
+    def SetAllIfacesUp
       interfaces = GetAllInterfaces()
 
       return false if interfaces.empty?
 
-      interfaces.all? { |i| SetLinkUp(i) }
+      interfaces.all? { |i| SetIfaceUp(i) }
     end
 
-    # Tries to set the link down of all available interfaces
+    # Tries to set all interfaces down
     #
     # @return [boolean] false if some of interfaces cannot be set down
-    def SetAllLinksDown
+    def SetAllIfacesDown
       interfaces = GetAllInterfaces()
 
       return false if interfaces.empty?
 
-      interfaces.all? { |i| SetLinkDown(i) }
-    end
-
-    # Calls wicked ifdown if it is the current backend, set the link down
-    # otherwise
-    #
-    # @param [String] name of interface to put down
-    # @see SetLinkDown
-    def SetInterfaceDown(dev_name)
```

Comment:
The idea behind this was to maintain compatible with iptools2 in case that wicked was not installed and `ifdown` not used replaced by `ip link set ` as we was using until now.

But I'm fine just doing an ifdown ang giving that responsibility to wicked.